docs/arcaflow/workflows/expressions.md

```diff
@@ -260,6 +260,7 @@ Functions:
 | `toLower(string)`                                | string       | Returns the input with Unicode letters mapped to their lower case.                                                                                                                                                                                                                                                                                                                                                                                                                             |
 | `toUpper(string)`                                | string       | Returns the input with Unicode letters mapped to their upper case.                                                                                                                                                                                                                                                                                                                                                                                                                             |
 | `splitString(string, string)`                    | list[string] | Returns a list of the substrings which appear between instances of the specified separator; the separator instances are not included in the resulting list elements; adjacent occurrences of separator instances as well as instances appearing at the beginning or ending of the input will produce empty string list elements.<br>&nbsp; Param 1: The string to split.<br>&nbsp; Param 2: The separator.                                                                                     |
+| `readFile(string)`                    | string | Given a string containing a file path, returns the contents of the indicated file as a UTF-8 character string. When passing a relative filepath, it must be written to be relative to the working directory from which the arcaflow process is executed. Additionally, shell environment variables are not expanded.                                                                                     |
 
```

Comment:
The first column should be a bit wider to match the column in the other rows.

I think we can make the sentence smoother by rewording the beginning of the second sentence to be:
> When passing a relative file path, the **path must be relative** to the working directory